src/pages/Home/Map.js

```diff
@@ -155,6 +164,38 @@ class Map extends Component {
     DisplayDraggablePin = () => this.setState({ DisplayDraggablePin: true });
     removeDraggablePin = () => this.setState({ DisplayDraggablePin: false });
 
+    /* Search functions BEGIN */
+    handleViewportChange = viewport => {
+        this.setState({
+          viewport: { ...this.state.viewport, ...viewport }
+        });
+      };
+    
+      // if you are happy with Geocoder default settings, you can just use handleViewportChange directly
+    handleGeocoderViewportChange = viewport => {
+    const geocoderDefaultOverrides = { transitionDuration: 1000 };
+
+    return this.handleViewportChange({
+        ...viewport,
+        ...geocoderDefaultOverrides
+    });
+    };
+    
+    handleOnResult = event => {
+        console.log(event.result);
```

Comment:
Unless this provides crucial information to the developer. I suggest removing console.log

Reply:
Removed console.log.